sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java

```diff
@@ -252,6 +253,13 @@ private CorePropertyDefinitions() {
         .defaultValue(Boolean.toString(false))
         .category(CATEGORY_ORGANIZATIONS)
         .type(PropertyType.BOOLEAN)
+        .build(),
+      PropertyDefinition.builder(ORGANIZATIONS_CREATE_PERSONAL_ORG)
```

Comment:
this would require to add dedicated WS to update them, which we don't have time to do just now

but, yes, they should definitely be internal properties